features/other_a/invite_request.feature

```diff
@@ -62,6 +62,18 @@ Feature: Invite requests
       And I should not see "Sorry, you have no unsent invitations right now."
       And I should see "You have 2 open invitations and 0 that have been sent but not yet used."
 
+  Scenario: User can fail to send an invitation to an invalid email address
+
+    Given invitations are required
+      And I am logged in as "user1"
+      And I request some invites
+      And an admin grants the request
+      And I try to invite a friend from my user page
+    When all emails have been delivered
+      And I fill in "invitation[invitee_email]" with "test@"
```

Comment:
When possible, we [prefer the user-facing label text](https://github.com/otwcode/otwarchive/wiki/Automated-Testing#our-cucumber-standards).